awslabs/aws-orbit-workbench
# ISSUE-51 adding sample notebooks

### Description:

Adding sample notebooks.

### Issue Reference URL

https://github.com/awslabs/aws-eks-data-maker/issues/51

Test instructions:

1. Start the environment
2. Open Jupyter notebooks instance
3. Open Example 1 and run it.
4. After running for some time, the notebook supposed to finish successfully.
5. Check the output for nested notebooks.

Note: 
- if nested failed, `Example 1` must fail as well.
- output location: <s3_output_bucket>/lake-creator/output/notebooks/tests/A-LakeCreator


----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS EKS Data Maker.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there a ISSUE associated with this PR?
- [x] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically 'main')?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0? 
- [ ] If applicable, have you updated the LICENSE file?
- [ ] If applicable, have you added the LICENSE header to new files?
